acceptance/setup/git/pre-suite/030_PuppetMasterSanity.rb

```diff
@@ -1,6 +1,6 @@
 test_name "Puppet Master sanity checks: PID file and SSL dir creation"
 
-pidfile = '/var/lib/puppet/run/master.pid'
+pidfile = '/var/run/puppetlabs/agent/master.pid'
```

Comment:
ah, so this is just the webrick pid file? what about `/var/run/puppetlabs/agent/puppet.pid` for the agent and `/var/run/puppetlabs/master/puppet.pid` for the master?